src/components/Admin/ManageQuestions.js

```diff
@@ -145,12 +146,32 @@ const ManageQuestions = ({ firebase }) => {
 
         await firebase
           .question(uid)
-          .delete()
-          .catch((err) => {
```

Comment:
We should start phasing out catch statements that set an error (i.e. we want to show an error to the user) as the error boundary will do that for us (however if we want a pretty error message instead of just the stack trace, we can throw a custom error like a few lines below)